sources/core/objects/turfs/MetalWall.h

```diff
@@ -1,26 +1,63 @@
 #pragma once
 
 #include "Turf.h"
+#include <bitset>
+#include <array>
 
 namespace kv
 {
 
+const std::array<Dir, 4> WALL_PROCESSING_DIRS = {Dir::NORTH, Dir::SOUTH, Dir::WEST, Dir::EAST};
+
+inline FastDeserializer& operator>>(FastDeserializer& file, std::bitset<4>& bset)
```

Comment:
It would be nice to add tests for `std::bitset` serialization, because usually serialization in the thing which causes most errors. It is needed to check that the result will be same for each platform and etc.